02-household-queries/requirements.in

```diff
@@ -3,9 +3,9 @@
 # > pip-compile requirements.in
 
 beautifulsoup4
-chainlit
+chainlit==1.0.500
 chromadb
-dspy-ai
+dspy-ai==2.4.3
```

Comment:
I had to pin to these specific versions to resolve some dependency version conflicts.